src/cromshell/cost/command.py

```diff
@@ -0,0 +1,482 @@
+import logging
+import statistics
+from datetime import datetime, timedelta, timezone
+
+import click
+from google.cloud import bigquery
+from tabulate import tabulate
+
+import cromshell.metadata.command as metadata
+import cromshell.utilities.command_setup_utils as command_setup_utils
+import cromshell.utilities.config_options_file_utils as cofu
+import cromshell.utilities.workflow_id_utils as workflow_id_utils
+import cromshell.utilities.workflow_status_utils as workflow_status_utils
+from cromshell import log
+from cromshell.utilities import io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="cost")
+@click.argument("workflow_ids", required=True, nargs=-1)
+@click.option(
+    "-c",
+    "--color",
+    is_flag=True,
+    default=False,
+    help="Color outliers in task level cost results.",
+)
+@click.option(
+    "-d",
+    "--detailed",
+    is_flag=True,
+    default=False,
+    help="Get the cost for a workflow at the task level",
+)
+@click.pass_obj
+def main(config, workflow_ids: str or int, detailed: bool, color: bool):
+    """
+    Get the cost for a workflow.
+    Only works for workflows that completed more than 24 hours ago on GCS.
+    Requires the 'bq_cost_table' key in the cromshell configuration file to be
+    set to the big query cost table for your organization.
+
+    Costs here DO NOT include any call cached tasks.
```

Comment:
It would be useful to at least annotate which tasks were call cached.

Reply:
created issue ticket
https://github.com/broadinstitute/cromshell/issues/250